### PR TITLE
Truncate filename column

### LINKED ISSUE #129: Improve appearance of field name cell in explore page files list
We need to decide with @inodb how handle this, with truncation and toolip, wider column, smaller text, etc.
image

- [ ] make link to synapse so one can download a single file easily @onursumer 
- [ ] remove the path (just basename)
- [ ] would be good to be able to see extension
- [ ] truncate the name (maybe tooltip that allows you to mouse into it). Show first 10 and last 5 symbols with ellipses in between

### PR DESCRIPTION
**Pull request by onursumer**
Fix #129

Synapse link is still missing. Need to merge #134 which exposes the `synapseId` field.

filename_truncate

### CODE REVIEW

File: lib/helpers.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import _ from 'lodash';
 import { NextRouter } from 'next/router';
 import fetch from 'node-fetch';
+import pathParse from 'path-parse';
```
Comment (inodb) [inline lib/helpers.ts:4]: feels slightly overkill to include a path parser dependency to do `lastIndexOf('/')`? No big deal tho, maybe it'll be useful in the future

**Review by inodb — APPROVED** [review]
Thanks!

File: lib/helpers.ts (hunk 1 above)
Comment (onursumer) [reply lib/helpers.ts:4]: good point. actually, there is also builtin node path module, probably better to use them instead of this one.

Comment (onursumer) [reply lib/helpers.ts]: removed the `path-parse` dependency, using builtin node path package now.

File: components/FileTable.tsx [hunk 2]
```diff
@@ -80,10 +81,24 @@ export default class FileTable extends React.Component<IFileTableProps> {
                 selector: 'filename',
                 wrap: true,
                 sortable: true,
+                cell: (file: Entity) => {
+                    return (
+                        <Tooltip overlay={getFileBase(file.filename)}>
+                            <a
+                                target="_blank"
+                                href={`https://www.synapse.org/#!Synapse:${
+                                    /* TODO file.synapseId */ file.filename
```
Comment (onursumer) [inline components/FileTable.tsx:90]: we need to merge #134 to be able to refer to `file.synapseId`

**Pull request merged**